suite-native/device/src/hooks/useHandleDeviceConnection.ts

```diff
@@ -35,6 +36,7 @@ export const useHandleDeviceConnection = () => {
     const isNoPhysicalDeviceConnected = useSelector(selectIsNoPhysicalDeviceConnected);
     const isPortfolioTrackerDevice = useSelector(selectIsPortfolioTrackerDevice);
     const isOnboardingFinished = useSelector(selectIsOnboardingFinished);
+    const isDeviceRemembered = useSelector(selectIsDeviceRemembered);
```

Comment:
If I have some other device remembered, and disconnect non-remembered device (imagine standard wallet remembered, passphrase not), it doesn't work as expected. Instead of redirecting to dashboard with warning about disconnected device, I'm redirected to account detail and then after couple of seconds to the dashboard without any warning. My suspicion is, that the condition shouldBlockSendReviewRedirect is not working in that case because isDeviceRemembered returns true if the non-remembered device is disconnected and remembered device is selected instead. But I haven't checked yet.

Reply:
I was unable to reproduce that :/

Reply:
Check the video, I can reproduce it even without any remembered device, but I guess my description wasn't clear at all...
https://github.com/user-attachments/assets/cce8a55d-fed3-4a98-b869-f5972998a36b

Reply:
Now I got it! The device disconnection error is possible, as TRANSPORT_ERROR.UNEXPECTED_ERROR. Handling of this error type added.

Also, the warning alert is sometimes displayed before the navigation takes place. So when the navigation happens later, the alert is hidden. An extra timeout was added to display the alert.

Both fixed by 45d6143b98.

Reply:
Not a fan of timeouts but we can live with that here 👌